src/refactorings/convert-for-to-foreach/convert-for-to-foreach.test.ts

```diff
@@ -149,6 +154,86 @@ items.forEach(item => {
 }`,
         expected: `this.data[0].items.forEach(item => {
   console.log(item);
+});`
+      },
+      {
+        description: "for-of",
+        code: `const items = ['foo', 'bar', 'baz'];
+
+[cursor]for (const val of items) {
```

Comment:
By default, the cursor is at the top of the code snippet. It looked suspicious then, because you wouldn't expect the refactoring to work on the for-of while the cursor isn't on it exactly.

That change allowed me to identify we forgot to check the for-of was indeed selected before refactoring it.